UnitedIncome/components
# InVision Mode

### Description
> Please provide a brief description of your changes below.

InVision's callback breaks Percy.

This error throws with these changes, but the build continues so I see no harm in it.
```
DSM =>  ✖ More than 1 call to "configure" found in /Users/james/Documents/Dev/components/.storybook/config.js.
    There should be only 1 call to configure inside "initDsm" "callback" parameter.
    For further assistance visit https://support.invisionapp.com 

```

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Run both Percy and Invision commands.

### Comments
> Any additional comments that will aid with the review process.

I'm guessing InVision's callback doesn't run when `build-storybook` is executed. 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
